update: authentication guides cookie information (#2642)
vtexdocs/dev-portal-content: docs/guides/Getting-Started/authentication/api-authentication-using-user-tokens.md

---
title: "API authentication using user tokens"
slug: "api-authentication-using-user-tokens"
excerpt: "Learn how to authenticate VTEX API requests with user tokens and match cookie and header names to Admin or storefront audiences."
hidden: false
createdAt: "2020-01-15T18:58:34.836Z"
updatedAt: "2026-05-07T12:00:00.000Z"
---

> When a user successfully logs in to a VTEX store, VTEX ID issues a [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token) user token and stores it in an HTTP-only cookie. For **Admin** sessions, the cookie name is `VtexIdclientAutCookie`. For **webstore** (shopper) sessions, the cookie name is `VtexIdclientAutCookie_{accountName}` or `VtexIdclientAutCookie_{accountId}`.

Reviewer: What is this `accountId`? Does it appear anywhere else other than this cookie?

This comment I'm making is not essential to the article, I guess.

> For a period of 24 hours after its creation, the user token can be used to authenticate requests to VTEX APIs. Send it in an HTTP header whose name matches that cookie (same pattern as above, depending on audience).

Reviewer 2: Acho mais seguro colocar

> Send it in an HTTP header as cookies whose name matches that cookie (same pattern as above, depending on audience

User tokens allow for actions limited to their scope, which is defined according to the user who logged in:

* **Shopper:** Shoppers' tokens have permission to perform actions related to the shopping experience, such as viewing active products' information, placing orders, and viewing information of orders made under that same shopper profile. This token scope does not allow users to access the VTEX Admin panel or change logistics settings, for example.
* **Admin:** Administrative users' tokens allow for actions based on [License Manager roles](https://help.vtex.com/en/docs/tutorials/roles) attributed to them. This may include access to different Admin panel pages or the ability to edit different configurations, for example.

> User tokens and associated permissions are tied to the user who logged in. Each request must use credentials valid for the VTEX account where that operation runs. The account in the API hostname must align with the account the token was issued for.

Reviewer: ```suggestion
User tokens and associated permissions are tied to the user who logged in. Each request must use credentials valid for the VTEX account where that operation runs. The account in the API hostname must be the same as the account the token was issued for.
```
